deploy/chart/Chart.yaml

```diff
@@ -1,3 +1,4 @@
 apiVersion: v2
 description: A Helm chart for Kubernetes
```

Comment:
Is this description verbose enough?

Reply:
This is the original description from 2017. If we decide to [include the chart as part of the release](https://github.com/operator-framework/operator-lifecycle-manager/issues/829) (as opposed to only using it for templating internally) then it should definitely be improved, but I don't think it needs to be touched as part of this PR.